docs/team/lzher385.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Lim Yu Zher's Project Portfolio Page
+---
+
+### Project: gitGud
+
+gitGud is a desktop application for managing friends' gaming information. gitGud uses a simple Graphical User Interface (GUI) combined with an optimized Command Line Interface (CLI) to give you a smooth and efficient experience. If you can type fast enough, gitGud can get you contact management tasks done faster than traditional GUI-based applications.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to link a friend with a game he/she plays.
+  * What it does: allows the user to associate a friend in his contact list with a game that was previously recorded.
+  * Justification: This feature improves the product significantly because a user is able to note down what games his/her friends play.
+  
+
+* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
```

Comment:
I don't think this is a valid feature we are planning to implement as of now. Can perhaps leave this out for later?

Reply:
My bad it was a line from the original template that I forgot to delete. Thanks for pointing it out!